packages/desktop-client/src/components/modals/OpenIDEnableModal.tsx

```
@@ -0,0 +1,94 @@
+import { useState } from 'react';
+
+import { send } from 'loot-core/platform/client/fetch';
+import * as asyncStorage from 'loot-core/platform/server/asyncStorage';
+import { getOpenIdErrors } from 'loot-core/shared/errors';
+import { type OpenIdConfig } from 'loot-core/types/models/openid';
+
+import { useActions } from '../../hooks/useActions';
+import { theme, styles } from '../../style';
+import { Error } from '../alerts';
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)